src/jdk.compiler/share/classes/com/sun/tools/javac/code/Lint.java

```diff
@@ -435,6 +450,8 @@ public static EnumSet<LintCategory> newEmptySet() {
 
         /** Does this category support being suppressed by the {@code @SuppressWarnings} annotation? */
         public final boolean annotationSuppression;
+
+        public String alias;
```

Comment:
```suggestion
        public final String alias;
```
We are no longer storing LintCategory objects.